<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectTab</class>
 <widget class="QWidget" name="ProjectTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>Project / Dataset Settings</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px; font-weight: bold; color: #000000; padding: 5px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupDataFolder">
     <property name="title">
      <string>Dataset Folder</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox { font-weight: bold; }</string>
     </property>
     <layout class="QVBoxLayout" name="dataFolderLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="folderPathLayout">
        <item>
         <widget class="QLabel" name="lblFolderPath">
          <property name="text">
           <string>Path:</string>
          </property>
          <property name="minimumWidth">
           <number>80</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboFolderPath">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="placeholderText">
           <string>Select dataset folder or choose from recent...</string>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBrowseFolder">
          <property name="text">
           <string>...</string>
          </property>
          <property name="browse" stdset="0">
           <string>true</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lblFolderStatus">
        <property name="text">
         <string>No folder selected</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #999999; font-style: italic; padding-left: 4px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mainContentLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupStructure">
       <property name="title">
        <string>Folder Structure</string>
       </property>
       <layout class="QVBoxLayout" name="structureLayout">
        <item>
         <widget class="QTreeWidget" name="treeStructure">
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <property name="indentation">
           <number>20</number>
          </property>
          <property name="styleSheet">
           <string notr="true">QTreeWidget { font-family: monospace; font-size: 12px; }</string>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rightPanelLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupDetectedInfo">
         <property name="title">
          <string>Detected Information</string>
         </property>
         <layout class="QFormLayout" name="detectedInfoLayout">
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lblClassesLabel">
            <property name="text">
             <string>Classes:</string>
            </property>
            <property name="styleSheet">
             <string notr="true">font-weight: bold;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lblClasses">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblCalibLabel">
            <property name="text">
             <string>Calibration:</string>
            </property>
            <property name="styleSheet">
             <string notr="true">font-weight: bold;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lblCalibInfo">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblPtLabel">
            <property name="text">
             <string>PT Models:</string>
            </property>
            <property name="styleSheet">
             <string notr="true">font-weight: bold;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lblPtInfo">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lblOnnxLabel">
            <property name="text">
             <string>ONNX Models:</string>
            </property>
            <property name="styleSheet">
             <string notr="true">font-weight: bold;</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="lblOnnxInfo">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lblHefLabel">
            <property name="text">
             <string>HEF Models:</string>
            </property>
            <property name="styleSheet">
             <string notr="true">font-weight: bold;</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="lblHefInfo">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="lblDataYamlLabel">
            <property name="text">
             <string>data.yaml:</string>
            </property>
            <property name="styleSheet">
             <string notr="true">font-weight: bold;</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="lblDataYamlInfo">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupAutoPath">
         <property name="title">
          <string>Auto-Configured Paths</string>
         </property>
         <layout class="QFormLayout" name="autoPathLayout">
          <property name="verticalSpacing">
           <number>8</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lblAutoCalibLabel">
            <property name="text">
             <string>Calib Dir:</string>
            </property>
            <property name="styleSheet">
             <string notr="true">font-weight: bold;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lblAutoCalibPath">
            <property name="text">
             <string>-</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #333333;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblAutoOnnxLabel">
            <property name="text">
             <string>ONNX Dir:</string>
            </property>
            <property name="styleSheet">
             <string notr="true">font-weight: bold;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lblAutoOnnxPath">
            <property name="text">
             <string>-</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #333333;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblAutoHefLabel">
            <property name="text">
             <string>HEF Dir:</string>
            </property>
            <property name="styleSheet">
             <string notr="true">font-weight: bold;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lblAutoHefPath">
            <property name="text">
             <string>-</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #333333;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnApplyToTabs">
         <property name="text">
          <string>Apply to All Tabs</string>
         </property>
         <property name="minimumHeight">
          <number>40</number>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #1565c0; color: white; font-size: 14px; font-weight: bold; padding: 10px 30px; border-radius: 5px;</string>
         </property>
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="rightSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
